### ISSUE TITLE
Entities with incorrect placement of their parts

### ISSUE THREAD

**Issue by Ametis81** [WolvenKit/Cyberpunk-Blender-add-on#122]
When importing entities that have animated parts (etc, lids, doors), they are placed incorrectly. This can also be noticed directly in WK when previewing such entities

Import `locker_medium_neomilitary.ent` into Blender. That is how its gonna look like:

![изображение_2024-03-09_160335711](https://github.com/WolvenKit/Cyberpunk-Blender-add-on/assets/117985431/9acc3eed-9c43-4665-a993-9aab6838a30c)
As you can see the door has its own pivot point and it's different

That's how you can see such entities in WK:

![изображение_2024-03-09_160527441](https://github.com/WolvenKit/Cyberpunk-Blender-add-on/assets/117985431/94c2662b-3ac0-4986-94a1-fa6f4d7d04f7)

**Comment by Simarilius-uk** [comment]
thanks for the report, will take a look

**Comment by Ametis81** [comment]
Other examples:

```
safe_medium.ent
disposal_device_crate.ent
ammo_case_small.ent
common_suitcase_small.ent
crate_small
cupboard_small.ent
```

**Comment by Ametis81** [comment]
` locker_medium_neomilitary.ent`

If we use coordinates from this section on animated part (door) it will be in correct place

![изображение_2024-03-09_170358329](https://github.com/WolvenKit/Cyberpunk-Blender-add-on/assets/117985431/e9d14aa1-ecc8-4e95-82cb-5eaec5d1d2aa)

![image](https://github.com/WolvenKit/Cyberpunk-Blender-add-on/assets/117985431/5e284060-efe5-446a-a6a3-9cfdfc794299)

**Comment by Simarilius-uk** [comment]
![image](https://github.com/WolvenKit/Cyberpunk-Blender-add-on/assets/29640139/2e9a80c7-7f77-4ffb-8c8a-8152e0a7a2d8)
Fixed.

**Comment by Simarilius-uk** [comment]
Closed by dd18633.
